aiohttp/web_reqrep.py

```diff
@@ -629,10 +634,22 @@ def start(coding):
                     return
 
     def start(self, request):
+        warnings.warn('use .prepare(request) instead', DeprecationWarning)
```

Comment:
They are does interchangeable with a note: `.prepare()` will process signal handing for response sending but `.start()` will not.